File: generator/output/ts-model/city.model.ts
class CityModel {
  public id: number = 0;

  // province 
  public province: string;

  // deleted 
  public deleted?: number = 0;

  // city 
  public city: string;

  // join_name 
  public join_name: string;

  public created_at: string;

  public updated_at: string;
}

const CityModelFields = {
  id: "id",
  province: "province",
  deleted: "deleted",
  city: "city",
  join_name: "join_name",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {CityModel, CityModelFields};

File: generator/output/entityModel/CityModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Boolean;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* City
*/
@Data()
public class CityModel {
  public Long id = null;

  /**
  * 
  */
  public String province = null;

  /**
  * 
  */
  public Boolean deleted = null;

  /**
  * 
  */
  public String city = null;

  /**
  * 
  */
  public String join_name = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
